### PR TITLE
Set execution permission to created app on linux-aarch64

### LINKED ISSUE #845: app doesn't have execution permission (linux-aarch64)

### PR DESCRIPTION
**Pull request by jperedadnr**
Fixes #845

### CODE REVIEW

File: src/main/java/com/gluonhq/substrate/util/FileOps.java [hunk 1]
```diff
@@ -696,6 +698,21 @@ public static String createPathingJar(String classpath) throws IOException {
...
+    /**
+     * Set execution permissions to an executable file at the given path
+     *
+     * @param path to an executable file
+     * @throws IOException
+     */
+    public static void setExecutionPermissions(Path path) throws IOException {
+        Set<PosixFilePermission> perms = new HashSet<>();
+        perms.add(PosixFilePermission.OWNER_READ);
+        perms.add(PosixFilePermission.OWNER_WRITE);
```
Comment (johanvos) [inline src/main/java/com/gluonhq/substrate/util/FileOps.java:710]: why is this needed?

**Comment by jperedadnr** [comment]
Not needed. Exec permissions were removed from GA upload action.
